test/spanningtrees/kruskal.jl

```diff
@@ -40,3 +40,68 @@ push!(vec_mst, Edge(3, 4))
 push!(vec_mst, Edge(2, 3))
 
 @test mst == vec_mst
+
+#second test
+g2 = Graph(8)
```

Comment:
Looks great; thanks for sticking with it. I'll merge it tonight assuming it passes tests. :)